Content.Shared/IdentityManagement/Components/IdentityComponent.cs

```diff
@@ -43,11 +45,22 @@ public IdentityRepresentation(string trueName, Gender trueGender, string ageStri
         PresumedName = presumedName;
     }
 
-    public string ToStringKnown(bool trueName)
+    public string ToStringKnown(bool showTrueIdentity)
     {
-        return trueName
-            ? TrueName
-            : PresumedName ?? ToStringUnknown();
+        string? nameToShow;
+        string? jobToShow;
+        if (showTrueIdentity)
+        {
+            nameToShow = TrueName;
+            jobToShow = TrueJob;
+        }
+        else
+        {
+            nameToShow = PresumedName ?? ToStringUnknown();
+            jobToShow = PresumedJob;
+        }
+        jobToShow = jobToShow != null ? $" ({jobToShow})" : "";
+        return nameToShow + jobToShow;
```

Comment:
Another alternative might be to actually add a "Title" field to the `MetaDataComponent` - so then the client could decide how/if they wanted to display the title.

I think that's a pretty big change though. Not sure!

Or a bunch of refactoring/reshaping of the crew manifest system so clients maintain a client-side record of job titles/things to add to any identity representations? Let me know what you think!